src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1885,12 +1885,12 @@ internal enum ErrorCode
 
         // The following warnings correspond to errors of the same name, but are reported
         // when a definite assignment issue is reported due to private fields imported from metadata.
-        WRN_UnassignedThisAutoProperty = 8880,
-        WRN_UnassignedThis = 8881,
+        WRN_UnassignedThisAutoPropertyUnsupportedVersion = 8880,
```

Comment:
FYI, I added a bullet to test plan to remind to add those error codes to the UpgradeProject fixer

Reply:
I am optimistic that our use of 'RequiredLanguageVersion' may make the UpgradeProject fixer "just work" through this API:

https://github.com/dotnet/roslyn/blob/82e850673b9c0a8a848a9a37dcfa2377817b6362/src/Compilers/Core/Portable/Compilation/Compilation.cs#L3643

I do think it will be best to add the tests and whatever changes end up being necessary in a subsequent PR.